src/virtual-machine.js

```diff
@@ -448,12 +446,13 @@ class VirtualMachine extends EventEmitter {
     }
 
     /**
-     * Export project as a Scratch 3.0 JSON representation.
+     * Export project or sprite as a Scratch 3.0 JSON representation.
+     * @param {string=} optTargetId - Optional id of a sprite to serialize
      * @return {string} Serialized state of the runtime.
      */
-    toJSON () {
+    toJSON (optTargetId) {
         const sb3 = require('./serialization/sb3');
```

Comment:
What do you mean?

Reply:
oh the import line... we don't have a lint rule for that and explicitly do requires elsewhere in the file sometimes. Mostly it's been for async require purposes (like loading the different extensions). I assume this particular one was a performance optimization that was made..?

Reply:
Could be. I'm not sure how well it could act as a performance optimization since the VM has to deserialize SB3 files during project loading (which is handled by the same file) anyway.

Reply:
Yeah, FWIW, it was introduced as part of #1956 in this commit: https://github.com/LLK/scratch-vm/commit/bc5605cf807a2b8b9c1b41253e8e89baad0fa266

I don't think I'll update this line of code in this PR since it's not relevant to the gray question mark work.